CodeEdit/NavigatorSidebar/SourceControlNavigator/Model/ChangesModel.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ChangesModel.swift
+//  CodeEdit
+//
+//  Created by Nanashi Li on 2022/05/05.
+//
+
+import Foundation
+import Git
+
+public final class ChangesModel: ObservableObject {
```

Comment:
Could you add a comment above explain what this does and why is needed? The name of the model is not really descriptive either